src/style.css

```diff
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: space-between;
+  height: 100%;
+  justify-content: center;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  vertical-align: middle;
```

Comment:
vertical-align has no effect on this element since it’s not an inline or table-cell element.

tldr: you can remove this ;)